### PR TITLE
Replace dev-env with GM in the Hermes guide

### PR DESCRIPTION
**Pull request by harveenSingh**
Closes: #2189

## Description

______

### PR author checklist:
- [x] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules). 
- [x] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: guide/src/tutorials/local-chains/relay-paths/multiple-paths.md [hunk 1]
```diff
@@ -4,95 +4,82 @@ Hermes can relay packets over all current or future paths between the configured
 
 Follow the steps below to connect three chains together and relay packets between them:
 
-1. Paste the following configuration in the standard Hermes configuration file at `~/.hermes/config.toml`:
+add to gm.toml
+gm start
+gm hermes config
+gm hermes keys
+gm status
```
Comment (seanchen1991) [inline guide/src/tutorials/local-chains/relay-paths/multiple-paths.md:11]: These steps get rendered all in a single line, which I don't think is the intention. Are these steps meant to replace the numbered steps below?

**Review by seanchen1991 — COMMENTED** [review]
A couple of things to be addressed, but looks good for the most part 🙂

File: guide/src/tutorials/local-chains/gaia.md [hunk 2]
```diff
@@ -20,9 +20,11 @@ Run the `make` command to build and install `gaiad`
 
 ```shell
 cd ~/go/src/github.com/cosmos/gaia
-git checkout v4.2.1
+git checkout <latest-release-tag> (https://github.com/cosmos/gaia/releases)
```
[the remark this answers is not in the document]
Comment (harveenSingh) [reply guide/src/tutorials/local-chains/gaia.md:23]: Agreed. Will remove it and put it as a note

File: guide/src/tutorials/local-chains/gaiad-manager.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  extra_wallets = 2
+  gaiad_binary = "~/go/bin/gaiad"
+  hdpath = ""
+  home_dir = "$HOME/.gm"
+  ports_start_at = 27000
+  validator_mnemonic = ""
+  wallet_mnemonic = ""
+
+  [global.hermes]
+    binary = "$HOME/ibc-rs/target/debug/hermes" #change this path according to your setup
```
[the remark this answers is not in the document]
Comment (harveenSingh) [reply guide/src/tutorials/local-chains/gaiad-manager.md:87]: Yes, agreed. Will change it.

File: guide/src/tutorials/local-chains/relay-paths/multiple-paths.md (hunk 1 above)
Comment (harveenSingh) [reply guide/src/tutorials/local-chains/relay-paths/multiple-paths.md:11]: This seems to be a typo. Will fix it.

**Pull request merged**